parser/model/ddl.go

```diff
@@ -412,6 +422,18 @@ func (job *Job) Encode(updateRawArgs bool) ([]byte, error) {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		if job.MultiSchemaInfo != nil {
+			for _, sub := range job.MultiSchemaInfo.SubJobs {
+				// Only update the args of last executing sub-job.
+				if sub.Args == nil {
```

Comment:
sub.Args means it's not the last job?

Reply:
Only the running sub-job decodes the `RawArgs` to `Args`.